nodejs/aws-infra/experimental/autoscaling/autoscaling.ts

```diff
@@ -20,6 +20,8 @@ import * as utils from "./../../utils";
 
 export class AutoScalingLaunchConfiguration extends pulumi.ComponentResource {
     public readonly launchConfiguration: aws.ec2.LaunchConfiguration;
+    public readonly id: pulumi.Output<string>;
```

Comment:
it's common enough to go to the underlying resource just to get the ID.  So to make that simpler, i've moved to a pattern of exposing the 'id' directly on the rich wrapper as well.